src/main/java/cinema/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,33 @@
+package cinema.security;
+
+import cinema.model.User;
+import cinema.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CustomUserDetailsService implements UserDetailsService {
+    @Autowired
+    private UserService userService;
+
+    @Override
+    public UserDetails loadUserByUsername(String userName) throws UsernameNotFoundException {
+        User user = userService.findByEmail(userName);
+        org.springframework.security.core.userdetails.User.UserBuilder userBuilder;
```

Comment:
```suggestion
        UserBuilder userBuilder;
```
спробуй винести зайве в імпорти

Reply:
done